cmd/monitor/monitor.go

```diff
@@ -34,6 +34,8 @@ var (
 	zero               = big.NewInt(0)
 	observedPendingTxs historicalRange
 	maxDataPoints      = 1000
+	maxConcurrency     = 10
```

Comment:
Annoying comment (sorry) but I think it's worth the effort to add a small comment at the top of each global variable to explain its purpose. We have a lot of parameters and variables in  `poycli monitor` and I imagine myself in 6 months forgetting about how all of this work 😅

Annoying comment (sorry) but I think it's worth adding a little comment at the top of each global variable to explain its purpose. We have a lot of parameters and variables in `poycli monitor` and I can already see myself in 6 months, looking at this piece of code to fix a bug and not understanding anything anymore 😅